awx/main/models/unified_jobs.py

```diff
@@ -1218,12 +1218,17 @@ def _websocket_emit_status(self, status):
                     status_data['instance_group_name'] = self.instance_group.name
                 else:
                     status_data['instance_group_name'] = None
+            elif status in ['successful', 'failed', 'canceled']:
```

Comment:
@mabashian is the UI resilient to this field not showing up here in the case of e.g., `status=error` ?

Reply:
The UI shouldn't collapse when we hit a message like that we just won't be able to update that field.  The logic for the jobs list is [here](https://github.com/ansible/awx/pull/5968/files#diff-02aed5f19734ca0e82f923146ab885dbR123).  Basically, we check to see if `finished` is included in the message and if it is we'll update the row.  On the Jobs list in particular the only "problem" this would cause would be that the client-side sorting we're attempting to do may get out of sync with what a real API response would provide.  This is because we're putting the rows without a `finished` timestamp up at the top of the list.

We do have some logic over on the dashboard that uses this same array for determining whether or not a job has finished (prompting us to go out and fetch new rows).  Should `error` be a part of [this array](https://github.com/ansible/awx/pull/5968/files#diff-5c4a87eec8896be28053f47299364133R111)?  I think it probably should?

Reply:
See https://github.com/ansible/awx/pull/5968/commits/f2955d7bb307f1effd680ed7c0bc730a11af347d